.github/workflows/ci.yml

```
@@ -157,6 +157,36 @@ jobs:
       - name: Run ESLint server boundary checks
         run: pnpm --filter=@jovie/web run lint:server-boundaries
 
+  ci-knip:
+    name: Knip (dead code)
+    # Detect unused files, dependencies, and exports
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)